src/node.cc

```diff
@@ -4442,6 +4445,8 @@ inline int Start(uv_loop_t* event_loop,
   isolate->SetAbortOnUncaughtExceptionCallback(ShouldAbortOnUncaughtException);
   isolate->SetMicrotasksPolicy(v8::MicrotasksPolicy::kExplicit);
   isolate->SetFatalErrorHandler(OnFatalError);
+  isolate->SetAllowWasmCodeGenerationCallback(
+      contextify::ContextifyContext::AllowWasmCodeGeneration);
```

Comment:
Node.js core shouldn't have to depend on `contextify`. Please move `AllowWasmCodeGeneration` to `node.cc`.

Reply:
Conceptually, `AllowWasmCodeGeneration` is used for *all* contexts, not just contexts created by `contextify`. It feels natural to move this function (which itself doesn't use any contextify-specific machinery) to node.cc.

Sorry for going back and forth on this, I didn't realize @bnoordhuis suggested this way.

Reply:
i agree with timothy, but i'm going to wait for my other context pr to get merged before i work on this more

Reply:
IIRC, when I made that suggestion, it was only setting it for ContextifyContext objects.  It makes sense to move it back now.